tstest/natlab/natlab.go

```diff
@@ -191,17 +191,22 @@ type routeEntry struct {
 	iface  *Interface
 }
 
-// NewMachine returns a new Machine without any network connection.
-// The name is just for debugging and need not be globally unique.
-// Use Attach to add networks.
-func NewMachine(name string) *Machine {
-	return &Machine{name: name}
-}
+// A PacketHandler is a function that can process packets.
+type PacketHandler func(p []byte, dst, src netaddr.IPPort) (drop bool)
```

Comment:
how about return a specific action type instead so code looks like `return Accept` and `return Drop` instead of boolean literals which are basically always terrible? :)